tools/build/citool

```diff
@@ -64,7 +68,8 @@ def parseArgs():
     parser = argparse.ArgumentParser(description='tool for analyzing logs from CI')
     subparsers = parser.add_subparsers(title='available commands', dest='cmd')
 
-    parser.add_argument('job', help='job number')
+    parser.add_argument('job', help='job number; for matrix jobs add the matrix index after a period')
```

Comment:
Possibly add an example `job number; for matrix jobs add the matrix index after a period e.g. 402061951.2`